docs/quick-start/starknet-devnet-with-asdf.mdx

```diff
@@ -0,0 +1,54 @@
+# StarkNet Devnet - Installation and Setup with asdf
+
+This document provides a guide to install and run **StarkNet Devnet** using `asdf`, a Rust-based tool for developers. Below, you'll find instructions for installation using `asdf`.
+
+## Requirements
+
+- **asdf**: Install `asdf` by following the [official asdf documentation](https://asdf-vm.com/).
+
+## Installation Using `asdf`
+
+1. **Add the StarkNet Devnet plugin**:
+
+   ```bash
+   asdf plugin add starknet-devnet
+   ```
+
+2. **List all available versions**:
+
+   ```bash
+   asdf list-all starknet-devnet
+   ```
+
+3. **Install the latest version**:
+
+   ```bash
+   asdf install starknet-devnet latest
+   ```
+
+4. **Set the installed version as global or local**:
+   ```bash
+   asdf global starknet-devnet <VERSION>
+   # or
+   asdf local starknet-devnet <VERSION>
+   ```
+
+## Running StarkNet Devnet
+
+After installation, you can run StarkNet Devnet with the following command:
+
+```bash
+starknet-devnet
+```
+
+## Additional Resources
+
+- [StarkNet Official Documentation](https://starknet.io/docs)
+- [StarkNet Devnet GitHub Repository](https://github.com/starknet/starknet-devnet)
```

Comment:
The link to the Starknet Devnet Repository is this

https://github.com/0xSpaceShard/starknet-devnet-rs


We can also leave a direct reference to the Starknet Devnet versions 

https://github.com/0xSpaceShard/starknet-devnet-rs/tags